### ISSUE TITLE
License Error

### ISSUE THREAD

**Issue by drewmassey-bah** [Esri/palladio#108]
When I load Houdini after installing the plug-in, I get a [pld] [fatal] The licensing system could not be initialized, please verify your 'LicParams' data.  Using Houdini 17.5.173 and CityEngine 2018.1.

**Comment by mistafunk** [comment]
Which palladio version did you try? Sent from my Huawei Mobile-------- Original Message --------Subject: [Esri/palladio] License Error (#108)From: drewmassey-bah To: Esri/palladio CC: Subscribed When I load Houdini after installing the plug-in, I get a [pld] [fatal] The licensing system could not be initialized, please verify your 'LicParams' data.  Using Houdini 17.5.173.

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/Esri/palladio/issues/108",
"url": "https://github.com/Esri/palladio/issues/108",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by drewmassey-bah** [comment]
I was pulling the newest from the GitHub (I think 1.6.0, and following the build process.  I modified the conanfile-h175.py to use the CityEngine SDK version 1.10.4051, and updated the file path for the CMAKE_PREFIX_PATH to reflect the path that was being created (added an additional esri_ce_sdk directory).  I was able to get a copy working once I was able to find the release repo, and copying the files over.
